docs/angular/guides/configuration.md

```diff
@@ -283,6 +283,21 @@ Tasks runners can accept different options. The following are the options suppor
 
 You can configure `parallel` and `maxParallel` in `nx.json`, but you can also pass them in the terminal `nx run-many --target=test --parallel`.
 
+### Workspace Layout
+
+You can add a `workspaceLayout` property to modify where libraries and apps are located.
+
+```json
+{
+  "workspaceLayout": {
+    "appsDir": "demos",
+    "libsDir": "packages"
+  }
+}
+```
+
+These settings would store apps in `/demos/` and libraries in `/packages/`.
```

Comment:
should we mention that the paths are always considered relative from the workspace root? 🤔